main/lxterminal/APKBUILD

```diff
@@ -1,22 +1,15 @@
 # Contributor: Natanael Copa <ncopa@alpinelinux.org>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=lxterminal
-pkgver=0.3.0
-pkgrel=2
+pkgver=0.3.1
+pkgrel=0
 pkgdesc="Desktop-independent VTE-based terminal emulator"
-url="http://lxde.org/"
+url="https://lxde.org/"
 arch="all"
 license="GPL-2.0-or-later"
-depends=""
 makedepends="vte-dev"
 subpackages="$pkgname-doc $pkgname-lang"
-source="https://downloads.sourceforge.net/lxde/$pkgname-$pkgver.tar.xz
-	CVE-2016-10369.patch"
-builddir="$srcdir"/$pkgname-$pkgver
```

Comment:
@prspkt is adding `builddir=...` and you are removing them. Maybe you two should talk about what you want so I don't need to review and merge "improvement" PRs which removes/adds `builddir=...` every 2 months. (see https://github.com/alpinelinux/aports/pull/4725#discussion_r202414862)

Reply:
`builddir=` should not be removed, period. I wonder why few people suddenly started removing them some time ago. I've already said some of them to stop doing it and reverted this particular change in many PRs before merging.

Another issue is removing secfixes. I don't understand why some people have this urge to remove them. :confused: